crates/sncast/tests/e2e/deploy.rs

```diff
@@ -84,8 +84,9 @@ async fn test_happy_case_human_output() {
             contract_address: [..]
             transaction_hash: [..]
 
-            Visit [..]
-            to see transaction details
+            Details:
```

Comment:
What about something like

```
See transaction (or some other word here) details:
  Contract: ...
  Transaction: ...
```

Just `Details:` is a bit "raw" compared to previous version with `Visit ... to see transaction details`.